### PR TITLE
add blogs

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <billmulligan516@gmail.com>

### CODE REVIEW

File: content/blog-posts/2023-02-13-delve/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-02-13T17:00:00.000Z"
+title: "How debugging Go programs with Delve and eBPF is faster"
+Description: "Learn how eBPF is faster than ptrace for tracing"
```
Comment (vannyle) [inline content/blog-posts/2023-02-13-delve/index.md:4]:
```suggestion
description: "Learn how eBPF is faster than ptrace for tracing"
```

File: content/blog-posts/2023-02-13-securitycon/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+date: "2023-02-13T17:00:00.000Z"
+title: "CloudNativeSecurityCon 2023: Identifying Suspicious Behaviors with eBPF"
+Description: "The eBPF wrap up from CloudNativeSecurityCon"
```
Comment (vannyle) [inline content/blog-posts/2023-02-13-securitycon/index.md:4]:
```suggestion
description: "The eBPF wrap up from CloudNativeSecurityCon"
```

**Review by vannyle — COMMENTED** [review]
The preview should be working after the fixes

**Comment by borkmann** [comment]
@xmulligan could you add an image for the delve one? At least the preview is not showing one.

File: content/blog-posts/2023-02-13-delve/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-02-13T17:00:00.000Z"
+title: "How debugging Go programs with Delve and eBPF is faster"
+description: "Learn how eBPF is faster than ptrace for tracing"
```
Comment (xmulligan) [inline content/blog-posts/2023-02-13-delve/index.md:4]:
```suggestion
description: "Learn how eBPF is faster than ptrace for tracing"
ogImage: ogimage.png
```

**Pull request merged**